scripts/ui/topological-ui-dev.sh

```diff
@@ -0,0 +1,32 @@
+BASE_DIR=~/Projects
+mkdir -p $BASE_DIR
+
+cd $BASE_DIR
+if [ ! -d insights-proxy ]; then
+  git clone git@github.com:RedHatInsights/insights-proxy.git
+  cd insights-proxy
+  npm install
+  sudo bash scripts/patch-etc-hosts.sh
```

Comment:
@martinpovolny these two scripts are missing

Reply:
but I don't know if I want to patch my etc/hosts. Is it necessary?

Reply:
they are not missing, they are part of the insight-proxy (see de `cd` command).

If you don't want to patch your `/etc/hosts`, then bad luck. You are not going to be able to develop the UI.